packages/contracts-bedrock/scripts/checks/test-names/main.go

```diff
@@ -1,170 +1,156 @@
 package main
```

Comment:
this was all generated by claude

Reply:
The original was? I changed it to use individual functions so that we can actually test each check